recipe/meta.yaml

```diff
@@ -1,9 +1,16 @@
 {% set name = "numpy" %}
 {% set version = "1.16.6" %}
-{% set mkl_random_version = "1.0.2" %}
-{% set mkl_random_buildnumber = 0 %}
-{% set mkl_fft_version = "1.0.6" %}
-{% set mkl_fft_buildnumber = 0 %}
+{% set mkl_random_version = "1.2.1" %}
+{% set mkl_random_buildnumber = 1 %}
+# this is the latest version of mkl_fft I could find that supported numpy 1.16 through testing.
+{% set mkl_fft_version = "1.2.1" %}
+{% set mkl_fft_buildnumber = 1 %}
+
+# When building out the initial package set for a new Python version the
+# recommendation is to build numpy-base but not numpy, then build
+# mkl_fft and mkl_random, and then numpy.
+# if full_build is false: build numpy-base only; otherwise build numpy-base and numpy
+{% set full_build = true %}
```

Comment:
I think that such configuration parameters should actually go to conda_build_config.yaml instead ... patching the recipe itself feels bad IMO.

Reply:
I updated this branch as well as master `numpy` to use `only_build_numpy_base: yes|no`. Let me know if that is cleaner. I had trouble getting a boolean value to work and be understood in our jinja rendering. So, I used what seemed like a common approach to use the string `'yes'`. I am open to trying some more things if that does seem okay.